plutus-ir/src/Language/PlutusIR/MkPir.hs

```diff
@@ -1,31 +1,28 @@
+{-# LANGUAGE FlexibleInstances     #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# OPTIONS_GHC -fno-warn-orphans #-}
```

Comment:
Move the instance to the file where `PIR.Term` is defined?
It's not a big deal either way.